modules/serverpod_auth/serverpod_auth_server/lib/src/business/users.dart

```diff
@@ -199,6 +216,17 @@ extension UserInfoMethods on UserInfo {
     return true;
   }
 
+  /// Updates the full name of this user, returns true if successful.
+  Future<bool> changeFullName(Session session, String newUserName) async {
+    if (id == null) return false;
+
+    var updatedUser = await Users.changeFullName(session, id!, newUserName);
```

Comment:
Suggestion: Fetch the user store the user Id and promote it locally to stay away from scary `!`.

```suggestion
    var userId = id;
    if (userId == null) return false;

    var updatedUser = await Users.changeFullName(session, userId, newUserName);
```

Reply:
Will fix. For further reference, is there a style guide that I should look up?

Reply:
I just copied, and altered when necessary, the the `changeUserName` method. So should I update that one as well?

Reply:
Yes please 🙏 

Unfortunately, I don't think we noted this in a style guide. We are actively working to remove these dangerous derefs so that we can add a linter rule for this in the future.